tests/util.py

```diff
@@ -114,3 +116,8 @@
 CERT_DNS_ONLY = load_certificate(FILETYPE_PEM, PEM_DNS_ONLY)
 CERT_CN_ONLY = load_certificate(FILETYPE_PEM, PEM_CN_ONLY)
 CERT_OTHER_NAME = load_certificate(FILETYPE_PEM, PEM_OTHER_NAME)
+
+backend = default_backend()
```

Comment:
these are strictly cryptography-only objects and should be moved into test_cryptography, no?